package.json

```diff
@@ -523,6 +523,14 @@
 					"command": "cosmosDB.refresh",
 					"when": "view == cosmosDBExplorer && viewItem == cosmosDBDocumentDatabase"
 				},
+				{
```

Comment:
nit: You could probably remove the refresh action from the 'cosmosDBDocumentCollection' item since it just has static nodes underneath it

Reply:
Yep, thanks. Originally docs were in this node, too.

Reply:
Actually as I user I'd rather have Refresh at all levels for consistency.